src/main/java/org/gridsuite/loadflow/server/repositories/LoadFlowResultRepository.java

```diff
@@ -105,7 +122,7 @@ public Optional<LoadFlowResultEntity> findResults(UUID resultUuid) {
     public void deleteAll() {
         globalStatusRepository.deleteAll();
         resultRepository.deleteAll();
-        limitViolationsRepository.deleteAll();
+        limitViolationRepository.deleteAll();
```

Comment:
Why didn't you remove this line like in delete function ? since the limitViolation will be deleted by deleting results